specs/eip4844/validator.md

```diff
@@ -29,25 +31,21 @@ This document represents the changes to be made in the code of an "honest valida
 
 ## Prerequisites
 
-This document is an extension of the [Bellatrix -- Honest Validator](../bellatrix/validator.md) guide.
+This document is an extension of the [Capella -- Honest Validator](../capella/validator.md) guide.
 All behaviors and definitions defined in this document, and documents it extends, carry over unless explicitly noted or overridden.
 
 All terminology, constants, functions, and protocol mechanics defined in the updated [Beacon Chain doc of EIP4844](./beacon-chain.md) are requisite for this document and used throughout.
 Please see related Beacon Chain doc before continuing and use them as a reference throughout.
 
 ## Helpers
 
-### `get_blobs_and_kzg_commitments`
+## Protocols
 
-The interface to retrieve blobs and corresponding kzg commitments.
+### `ExecutionEngine`
 
-Note: This API is *unstable*. `get_blobs_and_kzg_commitments` and `get_payload` may be unified.
-Implementers may also retrieve blobs individually per transaction.
+#### `get_payload`
 
-```python
-def get_blobs_and_kzg_commitments(payload_id: PayloadId) -> Tuple[Sequence[BLSFieldElement], Sequence[KZGCommitment]]:
```

Comment:
`get_blobs_and_kzg_commitments` is still mentioned below, do we want to remove this?

Reply:
Good catch. This was a bad merge from dev. I've added it back in. Thanks.